src/components/TodoItem/TodoItem.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import User from '../User/User';
+import { TodoItemProps } from '../../constants/proptypes';
+
+import './TodoItem.css';
+
+const TodoItem = ({ todo }) => {
+  const {
+    title, completed = false, user, id,
+  } = todo;
+
+  const classNames = !completed
+    ? 'todo-list-item'
+    : 'todo-list-item todo-list-item--done';
```

Comment:
https://npmjs.com/package/classnames (optional)

Reply:
what about this comment?